### ISSUE TITLE
saltutil.pillar_refresh fails

### ISSUE THREAD

**Issue by sebastian-philipp** [SUSE/sesdev#34]
```
    admin: ++ cat
    admin: ++ chown -R salt:salt /srv/pillar
    admin: ++ systemctl restart salt-master
    admin: ++ sleep 5
    admin: ++ salt '*' saltutil.pillar_refresh
    admin: ERROR: Minions returned with non-zero exit code
    admin: admin.two_node_ses7.com:
    admin:     Minion did not return. [No response]
    admin: node1.two_node_ses7.com:
    admin:     Minion did not return. [No response]
    admin: ++ sleep 2
    admin: ++ ceph-bootstrap config /Cluster/Minions add admin.two_node_ses7.com
    admin: /tmp/vagrant-shell: line 95: ceph-bootstrap: command not found
    admin: ++ ceph-bootstrap config /Cluster/Minions add node1.two_node_ses7.com
    admin: /tmp/vagrant-shell: line 96: ceph-bootstrap: command not found
    admin: ++ ceph-bootstrap config /SSH/ generate
    admin: /tmp/vagrant-shell: line 98: ceph-bootstrap: command not found
    admin: ++ ceph-bootstrap config /Containers/Images/ceph set registry.suse.de/devel/storage/7.0/cr/images/ses/7/ceph/ceph
    admin: /tmp/vagrant-shell: line 99: ceph-bootstrap: command not found
    admin: ++ ceph-bootstrap config /Time_Server/Server_Hostname set admin.two_node_ses7.com
    admin: /tmp/vagrant-shell: line 100: ceph-bootstrap: command not found
    admin: ++ ceph-bootstrap config /Time_Server/External_Servers add 0.pt.pool.ntp.org
    admin: /tmp/vagrant-shell: line 101: ceph-bootstrap: command not found
    admin: ++ ceph-bootstrap config /Deployment/Mon enable
    admin: /tmp/vagrant-shell: line 102: ceph-bootstrap: command not found
    admin: ++ ceph-bootstrap config /Deployment/Mgr enable
    admin: /tmp/vagrant-shell: line 103: ceph-bootstrap: command not found
    admin: ++ ceph-bootstrap config /Deployment/OSD enable
    admin: /tmp/vagrant-shell: line 105: ceph-bootstrap: command not found
    admin: +++ seq 1 3
    admin: +++ awk 'BEGIN{i=0; printf("[");}{if (i>0){printf(", ")}; i++; printf("\"/dev/vd%c\"", $1 + 97)}END{printf("]")}'
    admin: ++ DEV_LIST='["/dev/vdb", "/dev/vdc", "/dev/vdd"]'
    admin: ++ ceph-bootstrap config /Storage/Drive_Groups add 'value={"host_pattern": "node1*", "data_devices": { "paths": ["/dev/vdb", "/dev/vdc", "/dev/vdd"] }}'
    admin: /tmp/vagrant-shell: line 109: ceph-bootstrap: command not found
    admin: ++ ceph-bootstrap config /Deployment/Dashboard/username set admin
    admin: /tmp/vagrant-shell: line 111: ceph-bootstrap: command not found
    admin: ++ ceph-bootstrap config /Deployment/Dashboard/password set admin
    admin: /tmp/vagrant-shell: line 112: ceph-bootstrap: command not found
    admin: ++ ceph-bootstrap config ls
    admin: /tmp/vagrant-shell: line 114: ceph-bootstrap: command not found
    admin: ++ salt -G ceph-salt:member state.apply ceph-salt
    admin: ERROR: No return received
    admin: No minions matched the target. No command was sent, no jid was assigned.
Command '['vagrant', 'up']' failed: ret=1 stderr:
b"==> admin: An error occurred. The error will be shown after all tasks complete.\nAn error occurred while executing multiple actions in parallel.\nAny errors that occurred are shown below.\n\nAn error occurred while executing the action on the 'admin'\nmachine. Please handle this error then try again:\n\nThe SSH command responded with a non-zero exit status. Vagrant\nassumes that this means the command failed. The output for this command\nshould be in the log above. Please read the output to determine what\nwent wrong.\n"
```
a few seconds later, I did this manually:
```
(venv) ➜  two_node_ses7 vagrant ssh admin
vagrant@admin:~> sudo bash
admin:/home/vagrant # salt '*' test.ping
admin.two_node_ses7.com:
    True
node1.two_node_ses7.com:
    True
admin:/home/vagrant # salt '*' saltutil.pillar_refresh
admin.two_node_ses7.com:
    True
node1.two_node_ses7.com:
    True
admin:/home/vagrant # 
```
`

**Comment by smithfarm** [comment]
@sebastian-philipp Still having this problem?

**Comment by smithfarm** [comment]
Ah, it's a duplicate of https://github.com/SUSE/sesdev/issues/47

**Comment by sebastian-philipp** [comment]
nope. never happened since then to me

**Comment by smithfarm** [comment]
> nope. never happened since then to me

it does happen from time to time, but #52 should prevent it from breaking the deployment
